### PR TITLE
Move game speed lobby dropdown before time limit dropdown

### PR DESCRIPTION
**Pull request by dragunoff**
Game speed is the more frequently changed option and with the current layout it was buried below the fold.

Before this PR:
Screenshot from 2022-09-01 10-05-53

After this PR:
Screenshot from 2022-09-01 10-03-41

### CODE REVIEW

**Comment by PunkPun** [comment]
How is `DisplayOrder` organised in base mods? It seems that some things share the same order, do we want each option to have its own number?

**Comment by dragunoff** [comment]
> How is `DisplayOrder` organised in base mods? It seems that some things share the same order, do we want each option to have its own number?

The lobby options by default have a display order of '0' and then in the YAML the order is set. I guess if two traits share the same number then ordering depends on which trait is loaded first. For example:
```miniyaml
MapOptions:
	ShortGameCheckboxDisplayOrder: 2
	TechLevelDropdownDisplayOrder: 2
	GameSpeedDropdownDisplayOrder: 3
```

This kind of close sequential integer order is not great if we ever want to shuffle things around or if we want to allow maps to insert options with higher priority. Perhaps it'd be better to use some interval (say 10, 20, 30) so that options can be easily inserted in between.

**Comment by PunkPun** [comment]
If you do a search for `DisplayOrder` you'd notice that other options also have their own display order. I was referring to them all, not just ones in `MapOptions`.

Using interval of 10 sounds pretty good, it would be consistent with unit display order in production queue

**Comment by dragunoff** [comment]
> If you do a search for `DisplayOrder` you'd notice that other options also have their own display order. I was referring to them all, not just ones in `MapOptions`.
> 
Ah, yes - this was just an example of the `MapOptions` trait. There's a bunch of other traits of course. And some options are checkboxes, others are dropdowns.

> Using interval of 10 sounds pretty good, it would be consistent with unit display order in production queue

I'm suggesting it should start at `10` by default. This won't be consistent as most "displayOrder" fields are `0` by default. This is a part of a larger conversation I guess.

File: mods/d2k/rules/world.yaml [hunk 1]
```diff
@@ -225,7 +225,7 @@ World:
 		InnerSupportRadius: 3
 		OuterSupportRadius: 5
 	SpawnStartingUnits:
-		DropdownDisplayOrder: 1
+		DropdownDisplayOrder: 0
```
Comment (abcdefg30) [inline mods/d2k/rules/world.yaml]: If `0` is the default this node should be deleted. (Same for the other 0s.)

**Comment by dragunoff** [comment]
Update: addressed https://github.com/OpenRA/OpenRA/pull/20242#discussion_r960759980

**Review by PunkPun — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/6c246d69e428150eaca81bbb7911978146a7ee33...9b097cc3e16aaf8d7171ad951d777c7ca4e84bea)

**Pull request merged**